CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## [1.7.4](https://github.com/algolia/autocomplete/compare/v1.7.3...v1.7.4) (2022-12-20)
+
+
+### Bug Fixes
+
+* **insights:** add Algolia agent on `subscribe` ([#1058](https://github.com/algolia/autocomplete/issues/1058)) ([60f8ae4](https://github.com/algolia/autocomplete/commit/60f8ae46ae230c40be832b52da3e44dcdd204c58))
+* **react-renderer:** panel now closes when clicking/touching outside ([#1045](https://github.com/algolia/autocomplete/issues/1045)) ([b133e85](https://github.com/algolia/autocomplete/commit/b133e8577f84ec6c0f5007f559902b6a8490bad5))
```

Comment:
Actually this was only for examples, no changes to the code of the lib itself, maybe we should remove this line ?

Reply:
Ah right, this should have been `docs` instead of `fix` :) We'll be more careful next time.

Reply:
Addressed in [7189a77](https://github.com/algolia/autocomplete/pull/1059/commits/7189a77434fcd439e08ab1434894c89fa55464a8).